src/screens/HomeScreen.js

```diff
@@ -341,8 +345,8 @@ export const HomeScreen = ({ navigation }) => {
                           (mediaContent) =>
                             mediaContent.contentType === 'image' ||
                             mediaContent.contentType === 'thumbnail'
-                        )[0].sourceUrl &&
-                        !!eventRecord.mediaContents[0].sourceUrl.url,
+                        ).sourceUrl &&
+                        !!eventRecord.mediaContents.sourceUrl.url,
```

Comment:
here the `[0]` is gone for selecting the first element in `mediaContents`?

Reply:
yes first element in mediaContents should be there . But I still get the same warning and picture not visible . I have invested too much time in trying to solve this problem. I need help.

Reply:
bad !! in last line, see other comment